src/hooks/useWindowSize.tsx

```diff
@@ -0,0 +1,16 @@
+import { useLayoutEffect, useState } from 'react';
+
+function useWindowSize() {
```

Comment:
seems like you have hook to detect the device by its' width, do you need that new hook?

Reply:
yes, because this hook store the exact number of the width and height of the screen. The other hook only return true/false depends from the mobile or desktop resolution of the screen.

Reply:
I just see that you use this hook exactly for the same purpose as another one: understand if it is desktop or mobile, aren't you?

Reply:
in this case yes, but there is a difference in hooks logic and for now i would like to have both hooks for cases in implementation in a future.